web_external/main.js

```diff
@@ -12,6 +12,24 @@ import './stylesheets/item.comments.styl';
 import './routes';
 
 $(() => {
+    const gaKey = process.env.GA_KEY;
+    if (gaKey) {
+        // Inject the Google Analytics framework.
+        let s = document.createElement('script');
```

Comment:
@waxlamp Why aren't you using jQuery to do this DOM creation, lookup, and insertion?

Reply:
Because I like to avoid complexity when I don't understand it fully, and the vanilla JS way seemed nice and simple. Injecting a script tag via JQuery is not something I know all the possible subtleties of.

If that's a better way to do this, then let's update it in a new PR. But I thought we were going to switch this over to using the Girder GA plugin?